packages/contracts/contracts/Unirep.sol

```diff
@@ -676,18 +676,12 @@ contract Unirep is IUnirep, zkSNARKHelper, Hasher, VerifySignature {
             'Unirep: invalid public signals when verifyEpochKeyValidity'
         );
 
-        ProofsRelated memory proof;
-        // Unpack the snark proof
-        (proof.a, proof.b, proof.c) = unpackProof(input.proof);
-
         // Verify the proof
-        proof.isValid = epkValidityVerifier.verifyProof(
-            proof.a,
-            proof.b,
-            proof.c,
+        bool isValid = epkValidityVerifier.verifyProof(
```

Comment:
Could we return `epkValidityVerifier.verifyProof()` directly?

Reply:
yeah sounds good 👍🏻